src/software/ai/hl/stp/tactic/dribble/dribble_tactic_test.cpp

```diff
@@ -285,7 +285,7 @@ TEST_F(DribbleTacticTest, test_running_into_enemy_robot_knocking_ball_away)
     Point initial_position    = Point(-2, 1.5);
     Point dribble_destination = Point(-1, 2);
     Angle dribble_orientation = Angle::half();
-    BallState ball_state(Point(2, -2), Vector(1, 2));
+    BallState ball_state(Point(2, -2), Vector(2, 4));
```

Comment:
this was changed because the ball wasnt fast enough to knock into the enemy robot and make it out of their defense area